entity/src/sbom_package_license.rs

```diff
@@ -0,0 +1,57 @@
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, Eq, DeriveEntityModel)]
+#[sea_orm(table_name = "sbom_package_license")]
+pub struct Model {
```

Comment:
The one thing that worries me about this PR is that sbom_package_license seems to be very similar to purl_license_assertion / cpe_license_assertion do we need all 3?

Reply:
I don't think we need the purl and cpe variant. The license is tied to the package and the purl/cpes are simply additional names for those packages.

Reply:
@ctron so are you suggesting about removing the purl and cpe variants, i.e. removing the `purl_license_assertion` and `cpe_license_assertion` table and just keep the `sbom_package_license` one?

Reply:
I agree with Jens' suggestion, but I do have a question regarding the process of querying a package's license. If we were to remove those two entities, users would need to first query all SBOMs containing the package, then retrieve the node ID, and finally query the license. On the other hand, if we keep these entities, users can directly query the license of a package, although this would result in multiple records across different SBOMs. Given that a package might have different license records in various SBOMs, removing these entities seems like a reasonable approach.

Reply:
https://github.com/trustification/trustify/discussions/1396